lib/internal/fs/promises.js

```diff
@@ -264,12 +256,17 @@ class FileHandle extends EventEmitterMixin(JSTransferable) {
     if (this[kLocked])
       throw new ERR_INVALID_STATE('The FileHandle is locked');
     this[kLocked] = true;
-
+    const {
+      newReadableStreamFromStreamBase,
+    } = require('internal/webstreams/adapters');
```

Comment:
The internal `require()` is actually just one map load (to see if the module is already loaded) + one property load (state check for circular dependencies) for modules that are already loaded, so I think it's fine to just inline these as they are followed by more expensive operations that will shadow the overhead of `require()`